index/field/docs_field.go

```diff
@@ -598,7 +598,7 @@ type docsFieldBuilder struct {
 	bfb    boolFieldBuilder
 	ifb    intFieldBuilder
 	dfb    doubleFieldBuilder
-	sfb    stringFieldBuilder
+	sfb    bytesFieldBuilder
```

Comment:
bfb

Reply:
There is already a `bfb` for bool types.